tests/pw/pages/frontend/vendor-dashboard/settings/store/vendor-store.page.ts

```diff
@@ -0,0 +1,27 @@
+import { BasePage } from '@pages/basePage';
+
+export default class VendorStoreSettingsPage extends BasePage {
+    minimumOrderAmountInputField() {
+        return this.page.locator('#min_amount_to_order');
+    }
+
+    maximumOrderAmountInputField() {
+        return this.page.locator('#max_amount_to_order');
+    }
+
+    updateSettingsButton() {
+        return this.page.locator(`//input[@name="dokan_update_store_settings"]`);
+    }
+
+    async enterMinimumOrderAmount(amount: string) {
+        await this.minimumOrderAmountInputField().fill(amount);
```

Comment:
A locator, such as an input field, can have multiple use cases, such as fill (clear and fill), type (type only), pressSequentially, typeAndWait (type and wait before typing further), or appending new text. Additionally, filling in the input might trigger alerts or AJAX requests that must be handled simultaneously. Creating separate methods for every possible scenario would overcomplicate the project.  
We will add the appropriate action based on the usages instead of creating method for all scenarios.